src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -0,0 +1,487 @@
+        cleanedName = "S";
+      } else if (cleanedName.equals("double")) {
+        cleanedName = "D";
+      } else if (cleanedName.equals("int")) {
+        cleanedName = "I";
+      } else if (cleanedName.equals("long")) {
+        cleanedName = "J";
+      } else if (cleanedName.equals("boolean")) {
+        cleanedName = "Z";
+      } else {
+        //non-primitive array types look like "[Lpackage.of.MyClass;"
+        cleanedName = "L" + cleanedName + ";";
+      }
+      return Class.forName(arrayPrefix + cleanedName);
+    }
+
+    return Class.forName(className);
+  }
+
+  /**
+   * Returns class of exception or Throwable.
+   *
+   * @param className name of class as <code>String</code>
+   * @return the <code>Class</code> object
+   * @exception ClassCastException if class cannot be cast to Throwable
+   * @exception ClassNotFoundException if class cannot be found
+   */
+  @SuppressWarnings("unchecked")
+  public static Class<? extends Throwable> toThrowableClass(
+      final String className)
+      throws ClassCastException, ClassNotFoundException {
+    final Class throwableClass = Throwable.class;
+    Class<?> cl = forName(className);
+    if (cl.isAssignableFrom(throwableClass)) {
+      return (Class<? extends Throwable>) cl;
+    } else {
+      throw
+        new ClassCastException("cannot cast " + className + " to Throwable");
+    }
+  }
+
+  /**
+   * Removes the package from the name of a class, returning the class name
+   * only.
+   *
+   * @param className name of the class as <code>String</code>
+   * @return <code>String</code> value of the class name without preceeding
+     package name
+   */
+  public static String removePackageFromName(final String className) {
+    int lastSeparator = className.lastIndexOf('.');
+    if (lastSeparator != -1) {
+      return className.substring(lastSeparator + 1);
+    } else {
+      return className;
+    }
+  }
+
+  /**
+   * <code>classMap</code> keeps a cache of created classes.
+   *
+   */
+  private static HashMap<String, Class<?>> classMap = new HashMap<>();
+
+  /**
+   * <code>makePublic</code> sets member flag to public.
+   *
+   * @param m the member to make public
+   */
+  private static void makePublic(final CtMember m) {
+    int modifier = m.getModifiers();
+    modifier = modifier & ~(javassist.Modifier.PRIVATE
+                            | javassist.Modifier.PROTECTED);
+    modifier = modifier | javassist.Modifier.PUBLIC;
+    m.setModifiers(modifier);
+  }
+
+  /**
+   * Sets class flag to public.
+   *
+   * @param c the class as to make public
+   */
+  private static void makePublic(final CtClass c) {
+    int modifier = c.getModifiers();
+    modifier = modifier & ~(javassist.Modifier.PRIVATE
+                            | javassist.Modifier.PROTECTED);
+    modifier = modifier | javassist.Modifier.PUBLIC;
+    c.setModifiers(modifier);
+  }
+
+  /**
+   * This forces the creation of an instance for a given class name. If the
```

Comment:
Observe in the comments - "This forces the creation of an instance for a given class name..."
It should be "This forces the creation of an instance for a given class.", without the "name".